### PR TITLE
[CHORE] Standardise Image Metadata

### PR DESCRIPTION
**Pull request by elias7905**
# Description

- Standardised all Images to resize to 4000%
- Seeds resized to 6000%
- Reorganised some of the assets to make it more uniform and updated the paths in the code.
- Added gif images for chicken and El pollo
- (May delete if requested) Changed the photo for Radish Pie (food item from beta that's currently unused) with an unused asset of it to differentiate it from Apple Pie.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] I have read the contributing guidelines and agree to the T&Cs
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by 0xSacul — APPROVED** [review]
Like always Elias, you are amazing ❤️

File: public/erc1155/images/504(1).png
Comment (elias7905) [inline public/erc1155/images/504(1).png:1]: Radish Pie (504) and Apple Pie (524) currently share the same image. In the files there's this unused image of Radish pie. Could we use this image to reflect for Radish Pie instead? (and maybe in the future when Radish Pie is officially re-released we could use back this asset instead) I'll overwrite the original image metadata for radish pie when given the green light

File: public/erc1155/images/504.png
Comment (elias7905) [inline public/erc1155/images/504.png:1]: Radish Pie (504) and Apple Pie (524) currently share the same sprite. There's an unused sprite of the Radish Pie so I thought we can update the sprite to differentiate the 2 assets. Currently the Radish Pie is not in use but if it ever gets re-released we could use back this asset again.

**Review by spencerdezartsmith — APPROVED** [review]
Great job @elias7905 👏🏼

**Pull request merged**